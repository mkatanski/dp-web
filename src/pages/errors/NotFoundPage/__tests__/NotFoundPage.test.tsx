import React from "react";
import TestRenderer from "react-test-renderer";

import NotFoundView from "../NotFoundPage";

describe("pages", () => {
  describe("errors", () => {
    describe("NotFoundPage", () => {
      it("should render NotFoundPage component", async () => {
        const tree = TestRenderer.create(<NotFoundView />);
        expect(tree).toMatchSnapshot();
      });
    });
  });
});
