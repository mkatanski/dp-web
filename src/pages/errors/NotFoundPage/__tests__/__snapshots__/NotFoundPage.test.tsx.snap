// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`pages errors NotFoundPage should render NotFoundPage component 1`] = `
<div>
  404 - Not Found
</div>
`;
